### PR TITLE
Added test to verify dedicated admin can not manage Splunkforwarders

### PR DESCRIPTION
**Pull request by aliceh**

### CODE REVIEW

File: pkg/e2e/operators/splunkforwarder.go [hunk 1]
```diff
@@ -39,4 +47,73 @@ var _ = ginkgo.Describe(splunkForwarderBlocking, func() {
 	checkUpgrade(helper.New(), "openshift-splunk-forwarder-operator",
 		"openshift-splunk-forwarder-operator", "splunk-forwarder-operator",
 		"splunk-forwarder-operator-catalog")
+
+	ginkgo.Context("splunkforwarders", func() {
+		ginkgo.It("dedicated admin should not be able to manage splunkforwarders CR", func() {
+			sf := makeMinimalSplunkforwarder("SplunkForwarder", "splunkforwarder.managed.openshift.io/v1alpha1", "osde2e-splunkforwarder-test-1")
+			err := dedicatedAaddSplunkforwarder(sf, h)
```
Comment (cblecker) [inline pkg/e2e/operators/splunkforwarder.go:54]: We should defer a cleanup on both of these: https://github.com/openshift/osde2e/blob/00cde4de6721e1b4c182afe85860e6eae8332eee/pkg/e2e/verify/user_webhook.go#L51-L54

Comment (aliceh) [reply pkg/e2e/operators/splunkforwarder.go:55]: Thanks for reviewing @cblecker! I updated the code accordingly.

**Comment by aliceh** [comment]
@cblecker I realized that the 
`GOFLAGS='' go get github.com/markbates/pkger/cmd/pkger
go: found github.com/markbates/pkger/cmd/pkger in github.com/markbates/pkger v0.17.1
pkger --include /Users/alice/osde2e/assets --include /Users/alice/osde2e/configs
make: pkger: No such file or directory
make: *** [pkger] Error 1
`
error that I am seeing when building code is likely because my local environment requires some settings that are not there. 
It will most likely run correctly in someone else's environment.

**Comment by cblecker** [comment]
/close
Closing as this was added in #635

**Pull request closed without merging**